client/src/components/UI/NavBar.jsx

```diff
@@ -14,29 +14,39 @@ const styles = {
   grow: {
     flexGrow: 1,
   },
+  button: {
+    marginRight: 20,
+  },
 };
 
-const ButtonAppBar = ({ classes, isLoggedIn }) => (
-  <div className="NavBar">
-    <AppBar position="static">
-      <Toolbar>
-        <Typography variant="h6" color="inherit" className={classes.grow}>
-          PacMan Pro
-        </Typography>
-        {isLoggedIn
-          ? (
-            <Button color="inherit">Leader Board</Button>
-          )
-          : <Button color="inherit" href="/auth/signin">login</Button>}
-      </Toolbar>
-    </AppBar>
-  </div>
-);
+const NavBar = ({ classes }) => {
+  const isLoggedIn = Object.prototype.hasOwnProperty.call(localStorage, 'jwt');
```

Comment:
Create context for this.

Reply:
Ok I will create context for entire login